### PR TITLE
add: `winetricks-git`

### PR DESCRIPTION
**Pull request by Xdavius**

### CODE REVIEW

File: packages/winetricks-git/winetricks-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+pkgver="20240105"
+pkgdesc="Winetricks is an easy way to work around problems in Wine"
+url="https://github.com/Winetricks/winetricks"
+source=("https://github.com/Winetricks/winetricks.git")
+sha512sums=('SKIP')
+buildeps=("make")
+maintainer=("Xdavius <xdavius@github.com>")
+
+package() {
+  cd "${_archive}"
```
Comment (Elsie19) [inline packages/winetricks-git/winetricks-git.pacscript:13]: No using `${_archive}`.

File: packages/winetricks-git/winetricks-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,15 @@
+pkgname="winetricks-git"
+gives="winetricks"
+breaks=("${gives}-deb" "${gives}-bin" "${gives}-app" "${gives}")
+pkgver="20240105"
+pkgdesc="Winetricks is an easy way to work around problems in Wine"
+url="https://github.com/Winetricks/winetricks"
```
Comment (Elsie19) [inline packages/winetricks-git/winetricks-git.pacscript:6]:
```suggestion
url='https://github.com/Winetricks/winetricks'
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/winetricks-git/winetricks-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+source=("https://github.com/Winetricks/winetricks.git")
+sha512sums=('SKIP')
+buildeps=("make")
```
Comment (Elsie19) [inline packages/winetricks-git/winetricks-git.pacscript:9]: That's not a correct array name, it should be `makedepends`.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/winetricks-git/winetricks-git.pacscript (hunk 3 above)
Comment (Xdavius) [reply packages/winetricks-git/winetricks-git.pacscript:9]: True 😅

**Comment by Xdavius** [comment]
@Elsie19 ok for review 🐼

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**